HotelBooking/Views/book_reservation_view.py

```diff
@@ -44,6 +45,11 @@
         'message': "Enter the number of days you want to reserve",
         'name': 'duration',
     }],
+    "accomodation": [{
+        'type': 'input',
+        'message': "Do you require accessibility accomodations? Y/N",
+        'name': 'accomodation',
```

Comment:
also just realized you spelled accommodation wrong.

Reply:
rip